src/dataplane-protocol/src/batch.rs

```diff
@@ -27,7 +32,9 @@ pub trait BatchRecords: Default + Debug + Encoder + Decoder {
     }
 }
 
-impl BatchRecords for DefaultBatchRecords {}
+pub type MemoryBatch = Vec<Record>;
```

Comment:
This is not really batch.  It's sets of records.  So it should been MemoryRecords

Reply:
`MemoryRecords` it is 👍